src/main/java/au/gov/amsa/fgb/internal/ELTDTLocation.java

```diff
@@ -0,0 +1,154 @@
+package au.gov.amsa.fgb.internal;
+
+import com.github.davidmoten.guavamini.Lists;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+public class ELTDTLocation
+    extends BeaconProtocol {
+
+    public static final String ELTDT_PROTOCOL_CODE = "1001";
+    public static final String DEFAULT_COARSE_LATITUDE = "011111111";
+    public static final String DEFAULT_COARSE_LONGITUDE = "0111111111";
+
+    public ELTDTLocation() {
+        super(Lists.newArrayList("00", "10", "?0"), "ELT - ELT-DT Location Protocol", "110");
```

Comment:
what about simply `Arrays.asList` no need to add extra library dependencies

Reply:
Code was copied from another class and the dependency was already there. I didn't even notice this, well spotted. Will fix.